### ISSUE TITLE
Request: Improve Documentation on "SQL Like" Aliases in the UI

### ISSUE THREAD

**Issue by benyanke** [Mailu/Mailu#2045]
Currently, the alias wildcard interface looks like this:

![image](https://user-images.githubusercontent.com/4274911/140755729-b5fb52a4-60e4-4027-9daa-065c31117407.png)

From here: https://github.com/Mailu/Mailu/blob/master/core/admin/mailu/ui/forms.py#L141

The wildcard config is well documented here (https://mailu.io/master/webadministration.html?highlight=sql%20like#aliases), but it's not obvious looking directly in the UI.

Would you be open to a UI improvement that would be a bit more self-documenting?

I see a few potential options, which I'll outline with UI examples:

## Option 1: Provide multi-char example inline

Since presumably most users would be using multi-char wildcard, perhaps an inline example of that makes the most sense.

![image](https://user-images.githubusercontent.com/4274911/140757022-5ebfc0f7-9674-4710-8600-6e9f2107e583.png)

## Option 2: Link to docs

Another option would be a link to the documentation directly.

![image](https://user-images.githubusercontent.com/4274911/140756343-889c92ab-5cdd-48cc-ae8d-cade19794eda.png)

## Option 3: Both multi and single char wildcard inline

This seems the clunkiest, but it's an option as well.
![image](https://user-images.githubusercontent.com/4274911/140756946-722bc0e9-dd98-4bbc-90ce-a7c883ac0df5.png)

**Comment by nextgens** [comment]
I'd suggest we remove the tickbox altogether and assume that % is a wildcard

**Comment by benyanke** [comment]
I think that makes sense, but it might even be worth keeping something inline for it. Like this:

![image](https://user-images.githubusercontent.com/4274911/140778798-1e379be5-5995-4114-87ae-fc94ffc6b454.png)

**Comment by Diman0** [comment]
Valid characters for the local part are (From https://en.wikipedia.org/wiki/Email_address#Local-part):
- uppercase and lowercase Latin letters A to Z and a to z
- digits 0 to 9
- printable characters !#$%&'*+-/=?^_`{|}~
- dot ., provided that it is not the first or last character and provided also that it does not appear consecutively (e.g., John..Doe@example.com is not allowed).[5]

See https://mailu.io/master/webadministration.html#aliases
`_` and `%` are both relevant for SQL-Like syntax. 
`_` and `%` are also legal characters for the local part of an email address.

In other words, we need a checkbox so the user can clearly indicate that `_` and `%` can be treated as SQL-Like syntax.
We should keep the checkbox. Some users might use _/% in their normal email address which is legal.


Even though the message clearly states you can use SQL-Like syntax, we could perhaps add a link to the documentation that explains to the user what SQL-Like syntax is.

**Comment by benyanke** [comment]
That makes sense to me. So it sounds like we're centering on option 2 in the original post? I can make a PR providing that functionality, if folks agree this is the correct path forward.

![image](https://user-images.githubusercontent.com/4274911/140932832-9f6f1cf9-762d-4905-86d2-50e25a2b154e.png)

**Comment by nextgens** [comment]
Fine by me.

We look forward reviewing your PR :)
